### PR TITLE
Migrate remaining cloud-provider KEPs to new template

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #2220 

/assign @andrewsykim @cheftako

### CODE REVIEW

File: keps/sig-cloud-provider/providers/2532-huawei-cloud/kep.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+approvers:
+  - "@andrewsykim"
+  - "@cheftako"
+editor: TBD
+creation-date: 2019-12-06
+last-updated: 2020-02-26
+status: implementable
+see-also:
+replaces:
+superseded-by:
```
Comment (justaugustus) [inline keps/sig-cloud-provider/providers/2532-huawei-cloud/kep.yaml:18]:
```suggestion
```

We can drop the empty fields.

File: keps/sig-cloud-provider/providers/2532-huawei-cloud/kep.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+authors:
+  - "@RainbowMango"
+owning-sig: sig-cloud-provider
+reviewers:
+  - "@andrewsykim"
+  - "@cheftako"
+approvers:
+  - "@andrewsykim"
+  - "@cheftako"
+editor: TBD
```
Comment (justaugustus) [inline keps/sig-cloud-provider/providers/2532-huawei-cloud/kep.yaml:12]:
```suggestion
```

File: keps/sig-cloud-provider/providers/2531-baidu-cloud/kep.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+owning-sig: sig-cloud-provider
+reviewers:
+  - "@andrewsykim"
+  - "@cheftako"
+approvers:
+  - "@andrewsykim"
+  - "@cheftako"
+  - "@hogepodge"
+  - "@jagosan"
+editor: TBD
```
Comment (justaugustus) [inline keps/sig-cloud-provider/providers/2531-baidu-cloud/kep.yaml:14]:
```suggestion
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
@wojtek-t -- Just a few nits on empty fields.

Do we also want to add some symlinks to prevent external 404s?

File: keps/sig-cloud-provider/providers/2531-baidu-cloud/kep.yaml (hunk 3 above)
Comment (wojtek-t) [reply keps/sig-cloud-provider/providers/2531-baidu-cloud/kep.yaml]: done

File: keps/sig-cloud-provider/providers/2532-huawei-cloud/kep.yaml (hunk 2 above)
Comment (wojtek-t) [reply keps/sig-cloud-provider/providers/2532-huawei-cloud/kep.yaml]: done

File: keps/sig-cloud-provider/providers/2532-huawei-cloud/kep.yaml (hunk 1 above)
Comment (wojtek-t) [reply keps/sig-cloud-provider/providers/2532-huawei-cloud/kep.yaml]: done

**Review by wojtek-t — COMMENTED** [review]
@justaugustus  - Re symlinks - I don't see them being referenced anywhere (other than the issues, which I will update once this PR merges). So I don't think it's really needed.

**Comment by justaugustus** [comment]
> @justaugustus - Re symlinks - I don't see them being referenced anywhere (other than the issues, which I will update once this PR merges). So I don't think it's really needed.

SGTM!
/lgtm
/approve
/hold for SIG Cloud Provider

**Comment by andrewsykim** [comment]
/approve

Thanks @wojtek-t

**Comment by wojtek-t** [comment]
/hold cancel

[based on Andrew`s approve above]

**Pull request merged**